website-frontend/src/routes/research/overview/+page.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+	/** @type {import('./$types').PageData} */
+
+	import { PUBLIC_APIURL } from '$env/static/public';
+
+	export let data;
+
+	$: ({ laboratories } = data);
+</script>
+
+<div class="container h-full mx-auto flex-col justify-center items-center my-8">
+	<p>
+		The Department of Computer Science houses nine research laboratories catering to theoretical and
+		applied fields in computer science.
+	</p>
+
+	{#each laboratories as laboratory}
+		<div class="my-5 md:grid md:grid-cols-6 md:items-center py-5 border-b">
+			<div>
+				<img src="{PUBLIC_APIURL}assets/{laboratory.logo}" alt={laboratory.name} />
```

Comment:
`/` is missing after `{PUBLIC_APIURL}`. Kindly always note that we define URLs **without** trailing backslashes, just to avoid inconsistency issues in accessing asset URLs through environment variables.